### DISCUSSION TITLE
Does CWatM accepts monthly meteorological input and output?

### DISCUSSION THREAD

**Discussion by cele5425** [iiasa/CWatM#100] [Q&A] [answered]
Dear users
I am new to the CWatM, I would like to ask if the CWatM accepts monthly meteorological data to specifically calculate Evapotranspiration. If so which module to modify? Thank you !

**Comment by mikhailsmilovic** [comment, marked as answer]
Guten Tag, currently, CWatM accepts daily climatic inputs.
If locally available climate data is not available, there are global datasets at 30 arc minute resolution from ISIMIP, both for observed historical climate, and climate projections:

•	Observed climate data:
https://data.isimip.org/search/simulation_round/ISIMIP3a/product/InputData/climate_forcing/gswp3-w5e5/

•	Five climate models each with four RCP-SSP combinations available: https://data.isimip.org/search/tree/ISIMIP3b/InputData/climate/atmosphere/
